packages/commonwealth/server/workers/commonwealthConsumer/commonwealthConsumer.ts

```diff
@@ -63,7 +63,9 @@ function checkSubscriptionResponse(
 // properly handling/processing those messages. Using the script is rarely necessary in
 // local development.
 
-export async function setupCommonwealthConsumer(): Promise<void> {
+export async function setupCommonwealthConsumer(
+  skipRmqAdapter?: boolean,
+): Promise<void> {
   let brokerInstance: Broker;
   try {
     const rmqAdapter = new RabbitMQAdapter(
```

Comment:
do you need to create this when skip is true?

Reply:
I am not sure how the broker works, but from my point of view it just performs checks to ensure you are not instantiating the adapter twice.

We need to instantiate the adapter twice here because we are running these separate services in the same process (So I can get debugging and we won't need to rebuild the docker image each change)